x/poe/contract/contractio.go

```diff
@@ -176,6 +180,55 @@ func (a ContractAdapter) doExecute(ctx sdk.Context, msg interface{}, sender sdk.
 	return sdkerrors.Wrap(err, "execute")
 }
 
+// RawPageableResult is a query response type were the pagination key is the raw last element.
+type RawPageableResult interface {
```

Comment:
This interface is currently unused, for what I've seen. Do you think it makes sense to introduce it now?